tests/unit/test.config.js

```diff
@@ -120,8 +120,11 @@ describe('config', () => {
       const configObject = {
         foo: '/configured/foo',
```

Comment:
This test should not throw an error because it's testing that default values are preserved. You can fix it by changing `foo` to some kind of valid option. However, you will have to define a new option in addition to `source-dir` to do that.